instrumentation/ktor/ktor-3.0/testing/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+  id("otel.java-conventions")
+
+  id("org.jetbrains.kotlin.jvm")
+}
+
+val ktorVersion = "3.0.0"
```

Comment:
You have test classes under src/main so the tests really won't be executed. What you need to do is have the common testing classes in this module and move the tests elsewhere. I'd use the gradle test suite feature and create a suite for ktor3 under both ktor2 library and javaagent modules. Let me know if it is unclear how to do this.

Reply:
could you tell me if it would be simpler to just move tests to `src/test` source set?